keyboards/beatervan/beatervan.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+// There are a number of variations depending on the layout of your bottom row.
+// The arrow variant adds an additional key on the bottom-right, while the
+// command variant adds an additional key on the bottom-left. arrow-command is a
+// combination of both of those, having an additional key on both sides.
+//
+// Please note that the numbering of the macro arguments are based on the
+// numbers of the keys on the PCB.
+
+#define LAYOUT( \
+    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, \
+    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K169, \
+    K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, \
+    K300, K301, K302,           K303, K304,           K305, K306, K307  \
+) \
+{ \
+    { K000,   K001,   K002,   K003,  K004,   K005,   K006,   K007,   K008,   K009,   K010,   K011 }, \
+    { K100,   K101,   K102,   K103,  K104,   K105,   K106,   K107,   K108,   K109,   K110,   K169 }, \
+    { K200,   K201,   K202,   K203,  K204,   K205,   K206,   K207,   K208,   K209,   K210,   K211 }, \
+    { K300,   K301,   K302,  KC_NO,  K303,  KC_NO,   K304,   K305,   K306,   K307,   KC_NO }  \
+}
```

Comment:
```suggestion
    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, \
    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, \
    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, \
    K30, K31, K32,           K34, K36,           K37, K38, K39  \
) \
{ \
    { K00, K01, K02, K03,   K04,  K05,   K06,  K07, K08, K09, K0A,   K0B }, \
    { K10, K11, K12, K13,   K14,  K15,   K16,  K17, K18, K19, K1A,   K1B }, \
    { K20, K21, K22, K23,   K24,  K25,   K26,  K27, K28, K29, K2A,   K2B }, \
    { K30, K31, K32, KC_NO, K34,  KC_NO, K36,  K37, K38, K39, KC_NO, KC_NO }  \
}
```

Re-wrote to align with the indexing in the other layers -- it should be the same functionally.